bin/build-tasks-doc.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const got = require('got');
+const dedent = require('dedent');
+
+const npmUrl = 'https://registry.npmjs.com/'
+const taskList = require('../tasks.json');
+const tasksHeaderPath = path.join(__dirname, 'tasks-header.md');
+const tasksHeader = fs.readFileSync(tasksHeaderPath, 'utf8');
+const tasksOutputFilePath = path.join(__dirname, '..', 'docs', 'tasks.md');
+
+function catchError (err) {
```

Comment:
I think the functions in this file should be documented like we would in the cumulus repo